packages/wrangler/src/__tests__/vitest.setup.ts

```diff
@@ -149,19 +148,6 @@ vi.mock("../user/generate-random-state", () => {
 	};
 });
 
-vi.mock("xdg-app-paths", () => {
-	return {
-		__esModule: true,
-		default: vi.fn().mockImplementation(() => {
-			return {
-				config() {
-					return resolve("test-xdg-config");
-				},
-			};
-		}),
-	};
-});
```

Comment:
This change is notable because this library is no longer available from the Wrangler package. The code that is used from it is bundled into the workers-utils package.
So this mocking has no effect.